libr/core/cmd.c

```diff
@@ -2522,7 +2522,7 @@ static int cmd_kuery(void *data, const char *input) {
 		RLine *line = core->cons->line;
 		if (!line->sdbshell_hist) {
 			line->sdbshell_hist = r_list_newf (free);
-			r_list_append (line->sdbshell_hist, r_str_new ("\0"));
+			r_list_append (line->sdbshell_hist, strdup ("\0"));
```

Comment:
```suggestion
```

Reply:
i dont understand this line at all. i think is better to remove it. also "\0" is the same as "". so appending an empty line to the history should be the same as doing nothing. so lets remove the whole line and see if anything breaks fix it somewhere else